webapp/src/extensions.tsx

```diff
@@ -289,7 +289,7 @@ export class Extensions extends data.Component<ISettingsProps, ExtensionsState>
         const actions: sui.ModalButton[] = action ? [{ label: action, onclick: actionClick }] : undefined;
         if (!needsConsent && visible) this.initializeFrame();
         return (
-            <sui.Modal isOpen={visible} className={`${needsConsent ? 'extensionconsentdialog' : 'extensiondialog'}`}
+            <sui.Modal isOpen={visible} className={(needsConsent ? "extensionconsentdialog" : "extensiondialog")}
```

Comment:
did we need createModalClasses here?

Reply:
oh yes I thought it'd be good to take it away from extensions since minecraft doesn't have extensions (right?)

Reply:
Minecraft has extensions / you can still get to the dialog, it just doesn't have any approved extensions currently (which I think we want to change, we just need to find some time to think of some good ones to make and do it~)

Reply:
oo cool cool tyty i'll add it back

Reply:
okay so I added it to scriptsearch.tsx, which seems to be the modal that shows when we do the + extensions. I only dug a little into the modal where we ask for permissions - would you happen to know a case off the top of your head where that one shows up 😅 i will keep looking if not

Reply:
(for future reference) to test this modal, you can use this extension in micro:bit https://github.com/samelhusseini/pxt-midi-tone

(for today) i'm gonna not update this modal since I don't think there's a minecraft use for it, but we can always add in the future